stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -493,6 +517,26 @@ static String getRetrieveTokenApiUrl(@NonNull String tokenId) {
         return String.format("%s/%s", getApiUrl(), tokenId);
     }
 
+    static void handlePossibleErrors(StripeResponse response) throws
```

Comment:
baby baby nit: the "possible" in handlePossibleErrors doesn't seem informative. could it either be handle<WhatDoWeHandle>Errors or just handleErrors?

Reply:
Actually, this method can also do nothing -- it just checks for an error element in the response and throws the appropriate java exception if found. The "happy" path through the api does nothing in this method. I'll rename for clarification.